project/sites/imtac/settings.local.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
```

Comment:
This settings.local.php file should not be committed to git, so you'll need to remove it from this PR

Reply:
done, this is now removed from the commit